lib/ui/card_physics.dart

```diff
@@ -3,11 +3,12 @@ import 'package:flutter/cupertino.dart';
 const double CARD_WIDTH = (240 - 20) * (86.0 / 54) + 20;
 
 class CardPhysics extends ScrollPhysics {
-  final int cardCount;
-  const CardPhysics({ this.cardCount, parent }) : super(parent: parent);
+  final int? cardCount;
```

Comment:
Shouldn't be nullable

Reply:
Should be ? If it is set to non-nullable type, the following function calls will cause problems.